packages/packages.csv

```diff
@@ -156,7 +156,7 @@ Microsoft.Extensions.Configuration.Abstractions,ship,include
 Microsoft.Extensions.Configuration.AzureKeyVault,ship,include
 Microsoft.Extensions.Configuration.Binder,ship,include
 Microsoft.Extensions.Configuration.CommandLine,ship,include
-Microsoft.Extensions.Configuration.DockerSecrets,ship,include
+Microsoft.Extensions.Configuration.DockerSecrets,noship,include
```

Comment:
Yeah I had it as exclude but from https://github.com/aspnet/Coherence-Signed/wiki/New-package-Id-creation it sounds like that's a tool/framework bit since noship trumps that flag anyways